sdk/core/src/migration/credentials/ApiService.ts

```diff
@@ -16,6 +16,7 @@ interface FETCH_OPTIONS {
 
 const bodyRequiredMethods = ['POST', 'PATCH', 'PUT']
 
+// TODO: Why do we need this instead of using internal-api-clients?
```

Comment:
Already asked this question [in original PR](https://github.com/affinityproject/affinidi-core-sdk/pull/152#discussion_r727631511) (where it was marked as "resolved" without any response or changes) and on slack.

This is the only place that uses `node-fetch` directly. This entire class seems to duplicate the functionality of `tools-openapi`, and probably could have been implemented in a similar fashion to everything else in `internal-api-clients`.

So this code probably should be removed, but in order to do that we have to know why it exists in the first place.

Reply:
Refactored in #199